.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
   build:
     machine:
-      image: ubuntu-1604:201903-01 
+      image: ubuntu-1604:202010-01
```

Comment:
Need 1.27.0 that introduces `platform` param to docker compose v3
https://github.com/docker/compose/releases/tag/1.27.0-rc1